WordPress/Classes/Services/PushAuthenticationService.swift

```diff
@@ -9,15 +9,18 @@ import Foundation
 
 @objc public class PushAuthenticationService : NSObject, LocalCoreDataService
 {
+    var authenticationServiceRemote:PushAuthenticationServiceRemote?
+    
     /**
     *  @details     Designated Initializer
     *  @param       managedObjectContext    A Reference to the MOC that should be used to interact with
     *                                       the Core Data Persistent Store.
     */
     public required init(managedObjectContext: NSManagedObjectContext) {
+        super.init()
```

Comment:
Wow, i've missed the super.init call!. Not sure why Xcode didn't complain about it!